README.md

```diff
@@ -125,8 +126,7 @@ zip ABC.zip robot.py
 ## Doing a release
 
 1. Create a new tag & push
-2. Wait for CI to build a zip archive
-3. Upload the archive to the GitHub release for the tag
+2. Wait for CI to build a zip archive and upload it to the tag
```

Comment:
Not quite true. On top of pushing a tag, you also need to create a release in GitHub